oviewer/convert_align.go

```diff
@@ -84,6 +84,14 @@ func (a *align) convertDelm(src contents) contents {
 	dst := make(contents, 0, len(src))
 
 	s := 0
+	if indexes[0][0] == 0 {
+		if len(indexes) == 1 {
+			return src
+		}
+		dst = append(dst, src[0:indexes[0][1]]...)
+		s = indexes[0][1]
+		indexes = indexes[1:]
+	}
```

Comment:
This code is so cryptic 😅

I'm thinking about using this just after you checked indexes is not empty

firstEntry := indexes[0]

Then you could use a better name than `s` for firstEntry[0]

Also adding comments won't kill you you know.

My message is somehow a "clean your room, mom" post it 😬😂

Reply:
Thank you for your comment.
It was difficult to understand and had a problem, so I will change it significantly.